src/main/java/internetShop/db/DBConnectionHandler.java

```diff
@@ -0,0 +1,24 @@
+package internetShop.db;
+
+import internetShop.entity.product.Product;
+
+import java.sql.*;
+
+public class DBConnectionHandler {
+    private static final String URL = "jdbc:mysql://localhost:3306/store?serverTimezone=Europe/Moscow&useSSL=false";
+    private static final String USER = "root";
+    private static final String PASSWORD = "123";
+
+    public static void connect() {
+
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
It's unnecessary, maybe It was necessary until the 4th version of the driver came.